script/bigscreenplayer.js

```diff
@@ -82,58 +82,77 @@ define('bigscreenplayer/bigscreenplayer',
         return windowStartTime ? windowStartTime + (seconds * 1000) : undefined;
       }
 
-      return {
-        init: function (playbackElement, bigscreenPlayerData, newWindowType, enableSubtitles, newLiveSupport, device) {
-          Chronicle.init();
+      // Will need reusing in Player Component for Live CDN failover HLS
```

Comment:
Leave a TODO here at least if this needs to be done

Reply:
👍